client/client_test.go

```diff
@@ -214,7 +213,7 @@ func compareResults(t *testing.T, a, b client.Result) {
 	if a.Round() != b.Round() {
 		t.Fatal("unexpected result round", a.Round(), b.Round())
 	}
-	if bytes.Compare(a.Randomness(), b.Randomness()) != 0 {
+	if !bytes.Equal(a.Randomness(), b.Randomness()) {
```

Comment:
lol I wonder if it's even possible to use this `Compare` method then according to golint

Reply:
if you use compare as a comparator (e.g. are using the integer value for sorting greater than / less than) then it's allowed. if you're simply testing equality to 0 from the compare, prefer equal.